sdk/src/main/java/io/customer/sdk/CustomerIO.kt

```diff
@@ -545,17 +546,25 @@ class CustomerIO internal constructor(
 
     private fun recordScreenViews(activity: Activity, attributes: CustomAttributes) {
         val packageManager = activity.packageManager
-        return try {
-            val info = packageManager.getActivityInfo(
-                activity.componentName,
-                PackageManager.GET_META_DATA
-            )
-            val activityLabel = info.loadLabel(packageManager)
-
-            val screenName = activityLabel.toString().ifEmpty {
-                activity::class.java.simpleName.getScreenNameFromActivity()
+        try {
+            val screenName: String? = if (activity is TrackableScreen) {
+                // TrackableScreen takes precedence over manifest label
+                activity.getScreenName()
+            } else {
+                val info = packageManager.getActivityInfo(
```

Comment:
Thinking out load, we could have just exposed an option in the `meta data` of the manifest to skip these Activities which we don't intend to track.

Reply:
Interesting 🤔 I didn't think of it before. But having an interface gives more flexibility on determining the screen name, so let's stick with it.